templates/artist/includes/investor_profile_artist_list.html

```diff
@@ -8,11 +8,20 @@
 <div class="row small-up-1 medium-up-2 large-up-4 centered artistlist">
     {% for artist in artists.values %}
         <div class="column">
-            {% include "object_preview.html" with object=artist img=artist.photo.img %}
-            <p class="artist-details artist-invested">Total invested: ${{ artist.total_invested|floatformat:2|intcomma }}</p>
-            {% if artist.total_earned %}
-                <p class="artist-details artist-earned">Total earned: ${{ artist.total_earned|floatformat:2|intcomma }}</p>
-            {% endif %}
+            <div class="artist-list-image img-list">
+                {% include "object_preview.html" with object=artist img=artist.photo.img %}
+            </div>
+            <div>
+                {% if artist.total_earned %}
+                <p style="color: #4cbd7e;""><i class="fa fa-arrow-up" aria-hidden="true"></i>${{ artist.total_earned|floatformat:2|intcomma }} 
```

Comment:
I see what you are saying. I tried to just wrap the if statement around the the <p> but the else seems to be throwing things off.